File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int sum=0;
        int i=0;
        for (i=0;i<nums.size();i++){
            for(int j=0;j<nums.size();j++){
                if(i==j)
                continue;
            sum=nums[i]+nums[j];
            if(sum==target)
            return {i,j};
        }
        }
        return {};
    }
};

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap <Integer,Integer> mpp=new HashMap <>();
        for(int i=0;i<nums.length;i++){
           int current=nums[i];
            int m=target-current;
            if(mpp.containsKey(m))
            return new int []{mpp.get(m),i};
            mpp.put(current,i);
        }
        return new int []{1};
    }
}